pkg/controller.go

```diff
@@ -31,6 +31,11 @@ const (
 	healthPort = 9999
 
 	provisionerTimeout = "30s"
+
+	// reservation parameters
+	fastTimeoutEnv = "RESERVATION_FAST_TIMEOUT"
```

Comment:
Why that's strings in own variables ?

Reply:
What do you mean? They are constants

Reply:
What the point of this constants ? Why this variables are in constants and for example `NAMESPACE` not ?

Reply:
To clarify names of environment variables in a pod

Reply:
But I am not sure that I understand what we are talking about. What's up with Namespace here? It's passed from pod spec

Reply:
The main feature of constants is to share the same value and make code more DRY, you decide to move that strings in variable, what the point ? Does that strings also used somewhere in the code several times?

Reply:
To not use long string literals in the plain code is the default practice. They are used in csi-baremetal repo in controller also, so we can easily verify that they are the same